website/src/content/what-is-a-custom-application.mdx

```diff
@@ -14,9 +14,9 @@ A Custom Application is developed and managed by merchants.
 
 Using Custom Applications enables merchants to:
 
-* Extend new or existing functionalities of the commercetools Platform and the Merchant Center.
+* Extend new or existing functionalities of the Composable Commerce APIs and the Merchant Center.
 * Tailor specific business requirements and needs of merchants.
-* Integrate the commercetools Platform and the Merchant Center with external services.
+* Integrate the Composable Commerce APIs and the Merchant Center with external services.
```

Comment:
```suggestion
* Integrate Composable Commerce and the Merchant Center with external services
```

Reply:
@melindafekete thanks for the suggestion! What bothers me is that if we do it like that, it would be the only place where we separate Composable Commerce and the MC, since in other places we say "Composable Commerce APIs and the Merchant Center". What do you think?

Reply:
Good point, let's keep the original change 👍